.github/workflows/build-magma-rocm-linux.yml

```diff
@@ -0,0 +1,69 @@
+name: build-linux-magma-rocm
+
+on:
+  push:
+    branches:
+      main
+    paths:
+      - .ci/magma-rocm/*
+      - .ci/magma-rocm/package_files/*
+      - .github/workflows/build-magma-rocm-linux.yml
+  pull_request:
+    paths:
+      - .ci/magma-rocm/*
+      - .ci/magma-rocm/package_files/*
+      - .github/workflows/build-magma-rocm-linux.yml
+
+defaults:
+  run:
+    shell: bash -x -e -l {0}
+env:
+  BUILD_ENVIRONMENT: build-linux-magma-rocm
+  IN_CI: 1
+  IS_GHA: 1
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
+  cancel-in-progress: true
+
+jobs:
+  build-linux-magma-rocm:
+    if: github.repository_owner == 'pytorch'
+    runs-on: linux.2xlarge
+    permissions:
+      id-token: write
+    strategy:
+      matrix:
+        rocm_version: ["63", "624"]
```

Comment:
Can we somehow infer it from the rest of the pipeline (otherwill we'll likely to run into situation when those are out of sync with other builds)

Reply:
I was thinking we might be able to do our N and N-1 strategy but we still need to use the right docker image which has rocm version baked in as 6.3 etc.  Unless we also move to an N N-1 naming for our manylinux images we're stuck with this.  That can be evaluated later in a follow-up PR.